tests/contracts/example_contract_1.json

```diff
@@ -0,0 +1,64 @@
+{
+    "request":{
+        "url":"http://localhost/test",
+        "method":"POST",
+        "header": {{config.default_headers|jsonify}},
```

Comment:
Please leave spaces after and before double curly braces like {{ foo }} instead of {{foo}}